exercises/exercise4/checkSpecialCases.xquery

```diff
@@ -0,0 +1,149 @@
+xquery version "1.0";
+
+declare namespace local = "local";
+declare variable $gameInstance := fn:doc("GameInstance.xml")/mancalaGame;
+declare variable $currentPlayer := local:getCurrentPlayer();
+declare variable $oppositePit := local:getOppositePit($lastPit);
+
+(: This is the main function which triggers all other helper functions :)
+declare function local:checkSpecialCases($lastPit as xs:string) {
```

Comment:
Use REST to trigger the method. See how it's done in controller.xq and common.xq.